src/status_im2/navigation/roots.cljs

```diff
@@ -1,78 +1,26 @@
-                          :noBorder true}
-   :elevation            0
-   :title                {:color (colors/theme-colors colors/neutral-100 colors/white)}
-   :rightButtonColor     (colors/theme-colors colors/neutral-100 colors/white)
-   :background           {:color (colors/theme-colors colors/white colors/neutral-100)}
-   :backButton           {:color  (colors/theme-colors colors/neutral-100 colors/white)
-                          :testID :back-button}})
-
-(defn default-root
-  []
-  {:layout {:componentBackgroundColor (colors/theme-colors colors/white colors/neutral-100)
-            :orientation              "portrait"
-            :backgroundColor          (colors/theme-colors colors/white colors/neutral-100)}})
-
-(defn merge-top-bar
-  [root-options options]
-  (let [options (:topBar options)]
-    {:topBar
-     (merge root-options
-            options
-            (when (or (:title root-options) (:title options))
-              {:title (merge (:title root-options) (:title options))})
-            (when (or (:background root-options) (:background options))
-              {:background (merge (:background root-options) (:background options))})
-            (when (or (:backButton root-options) (:backButton options))
-              {:backButton (merge (:backButton root-options) (:backButton options))})
-            (when (or (:leftButtons root-options) (:leftButtons options))
-              {:leftButtons (merge (:leftButtons root-options) (:leftButtons options))})
-            (when (or (:rightButtons root-options) (:rightButtons options))
-              {:rightButtons (merge (:rightButtons root-options) (:rightButtons options))}))}))
+  (:require [status-im2.navigation.view :as views]
+            [status-im2.navigation.options :as options]
+            [status-im2.constants :as constants]))
 
 (defn get-screen-options
   [screen]
   (merge (get-in views/screens [screen :options])
-         (status-bar-options)
-         (merge-top-bar (topbar-options)
-                        (get-in views/screens [screen :options]))))
+         (options/statusbar-and-navbar-root)
+         (options/merge-top-bar (options/topbar-options)
+                                (get-in views/screens [screen :options]))))
 
-;;TODO problem here is that we have two places for screens, here and in screens ns, and we have handler
-;;in navigate
 (defn old-roots
   []
-  ;;TABS
   {;; ONBOARDING
    :onboarding
    {:root {:stack {:id       :onboarding
                    :children [{:component {:name    :get-your-keys
                                            :id      :get-your-keys
-                                           :options (status-bar-options)}}]
-                   :options  (merge (default-root)
-                                    (status-bar-options)
-                                    {:topBar (assoc (topbar-options)
+                                           :options (options/statusbar-and-navbar-root)}}]
```

Comment:
consider making a separate options for all of these old options so that we do not need to think about these when adjusting the other options.
e.g options/statusbar-and-navbar-root

Reply:
what old options? we don't have old options

Reply:
Ok so the old screens work exactly as the new ones? I thought the designs were a bit different on the old screens and so the old options are a bit different.
For example they use the inbuilt topNav elements etc.